ntbs-service/Pages/Search/Index.cshtml

```diff
@@ -9,12 +9,20 @@
 }
 
 <h1>Search</h1>
-<div>
+<div class="search-page">
     <form method="get">
         <div asp-validation-summary="ModelOnly"></div>
-        <span class="enter-search-details-tip">
-            To search enter at least one of the following fields
-        </span>
+        @{
+            var SearchParametersError = !Model.validationService.IsValid("SearchParameters");
+            var SearchParametersGroupState = SearchParametersError ? Error : Standard;
+        }
+        <nhs-form-group nhs-form-group-type=@SearchParametersGroupState>

```

Comment:
This can wrap all of the required fields if we want or we could add an error summary to the top of the page which could just contain this error

Reply:
Ooh, that might be a nice way to get there -can you show a screenshow of what the error state looks like?

Reply:
Oh, also we'll want an `aria-describe-by` attribute here linking to the error `span` (and perhaps to the `tip` too - incidentally, should that tip he a `hint` from the library?)

Reply:
![image](https://user-images.githubusercontent.com/37540516/66850646-2144a300-ef71-11e9-88b2-4c170e1d75f9.png)

![image](https://user-images.githubusercontent.com/37540516/66850697-3b7e8100-ef71-11e9-9f21-79383b04a28d.png)

One of these two kind of approaches could work?

Reply:
Seems to be good, the first I one in particular! :)